Sources/SPMTestSupport/XCTAssertHelpers.swift

```diff
@@ -131,7 +132,7 @@ public func XCTAssertBuildFails(
     line: UInt = #line
 ) -> CommandExecutionError? {
     var failure: CommandExecutionError? = nil
-    XCTAssertThrowsCommandExecutionError(try executeSwiftBuild(path, Xcc: Xcc, Xld: Xld, Xswiftc: Xswiftc), file: file, line: line) { error in
+    XCTAssertThrowsCommandExecutionError(try executeSwiftBuild(path, extraArgs: extraArgs, Xcc: Xcc, Xld: Xld, Xswiftc: Xswiftc), file: file, line: line) { error in
```

Comment:
nit: line length. If you're editing in Xcode, for simple function invocations like this one, most of the time Ctrl-M hotkey formats in the style we want.